cli/util.go

```
@@ -30,6 +30,9 @@ func ParseAddress(s string) (loom.Address, error) {
 	}
 
 	b, err := ParseBytes(s)
+	if err != nil {
+		return loom.Address{}, nil
```

This should return the error, not hide it